spec/validations_spec.rb

```diff
@@ -1,19 +1,151 @@
+require 'spec_helper'
 require_relative '../lib/validations'
+require_relative '../lib/ship'
 
 RSpec.describe Validations do
 
   let(:validations) {Validations.new}
+  let(:destroyer) {Ship.new("destroyer", 2)}
 
-  it "raises error for wrong input for ship number" do
-    output = validations.validate_ship_number("5")
+  describe "checks player's entered number corresponds to a ship" do
+    it "returns :invalid_ship_number for invalid number" do
+      output = validations.validate_ship_number("5")
 
-    expect(output).to eq(:error)
+      expect(output).to eq(:invalid_ship_number)
+    end
+
+    it "returns :invalid_ship_number for invalid input type" do
+      output = validations.validate_ship_number("hello")
+
+      expect(output).to eq(:invalid_ship_number)
+    end
+
+    it "returns :valid_ship_number for valid input" do
+      output = validations.validate_ship_number("2")
+
+      expect(output).to eq(:valid_ship_number)
+    end
+  end
+
+  describe "checks player's inputs for ship to place are all valid" do
```

Comment:
This is a bit hard to read. Maybe better to say `checks player's input for ship placement is valid`